templates/master/01-master-kubelet/_base/files/kubelet.yaml

```diff
@@ -32,6 +32,8 @@ contents:
       IPv6DualStack: true
 {{- end}}
       LegacyNodeRoleBehavior: false
+      # Will be removed in future openshift/api update https://github.com/openshift/api/commit/c8c8f6d0f4a8ac4ff4ad7d1a84b27e1aa7ebf9b4
+      RemoveSelfLink: false
```

Comment:
@deads2k with kube 1.20 bump, some of the kubelet-config unit tests were failing. Can you please let us know if the kubelet related changes in this PR are ok?

Reply:
@rphillips PTAL

Reply:
RemoveSelfLink is being removed in 1.21... I don't think we need this change.

Reply:
I retract that comment after more review. +1